### PR TITLE
trivy based BOM & BOV reports added

### PR DESCRIPTION
**Pull request by birkb**
- to make vulnerability tracking easier
- debian package list added too
- createBuildDir moved before installPrerequisites for the trivy install
- reports added to release assets
grafik

### CODE REVIEW

**Review by Joe-Public — DISMISSED** [review]
lgtm

In case local build time becomes an issue in the future, it might be an option to keep the trivy executable and the cache (`trivy --cache-dir ...`) inside the build dir and only download the binary again when there is a newer version (`trivy --version`). Or even an option to completely skip `generateBom` during a local build.

**Comment by birkb** [comment]
> lgtm
> 
> In case local build time becomes an issue in the future, it might be an option to keep the trivy executable and the cache (`trivy --cache-dir ...`) inside the build dir and only download the binary again when there is a newer version (`trivy --version`). Or even an option to completely skip `generateBom` during a local build.

We can also remove `generateBom` for `localBuild` because currently the trivy results will only be attached to the release. If nobody has a local use case we should remove it...

File: debian-live/build.sh [hunk 1]
```diff
@@ -490,17 +490,81 @@ function installPrerequisites {
...
   sudo chmod 755 /usr/local/bin/yq
   if [ "$?" -ne 0 ]; then
     logerror "${FUNCNAME[0]}" "yq permission update failed"
     exit 1
   fi
+
+  loginfo "${FUNCNAME[0]}" "yq installation done"
+}
+
+function downloadTrivy {
```
Comment (birkb) [inline debian-live/build.sh:515]: @Joe-Public i have added a `downloadTrivy` that checks if it is running on a Github runner...

Comment (Joe-Public) [reply debian-live/build.sh:515]: Great, I guess it's most important to have it for the official releases

**Review by Joe-Public — APPROVED** [review]
tested the local build and trivy gets not installed and generateBom is not done

**Pull request merged**